### ISSUE TITLE
Uncaught exeception makes Uwazi fail silently

### ISSUE THREAD

**Issue by txau** [huridocs/uwazi#4047]
In a Uwazi instance that has data coming from a migration from another system, which probably was injected directly in the database without proper validation, it happens that there are thesauri values in the entities that do not exist in the actual thesauri.

This triggers the following nuances:
- Since the values and labels are denormalized, when viewing the entity the values appear but when editing the multi-selects are empty.
- When attempting to save a validation error is triggered, but an uncaught exception happens during the process (as show in the screenshot) so there is no user feedback in the UI and also there is no server side logging of the error.
- Moreover, despite getting a "422 - unprocessable entity" error, the activity log shows an entry for the edition but the new data was not recorded.

![image](https://user-images.githubusercontent.com/489831/143265106-54df1e09-ec36-4f2c-8cca-9ce677303d69.png)

**Comment by LaszloKecskes** [comment]
@txau  This is about specific problem in a specific dataset, from almost two years ago. Should we still check it, or close it?

**Comment by txau** [comment]
@LaszloKecskes not sure. I don't remember the details. I was wondering if at least the uncaught exception should be handled?

**Comment by LaszloKecskes** [comment]
@txau Yes. Also, the issue describes it in such a way that it is easily reproducible without the original data. I've tried it out, it's still happening. So, you are right, let's keep this one open.
